matlab/scripts/prepare_worker.m

```diff
@@ -1,5 +1,11 @@
-r  = experiment.Session & 'session_date>"2016-02"';
+r  = experiment.Session & 'session_date>"2016-02"' & experiment.AutoProcessing;
+
 while true
-    parpopulate(preprocess.Prepare, r)
+    while count((preprocess.Prepare().popRel & r) - preprocess.Prepare)
```

Comment:
I'll add it. I actually checked for that but progress is missing that in it's help string.